blazor/pivot-table/pivot-chart.md

```diff
@@ -1028,6 +1028,96 @@ In the following code sample, the pivot chart can be seen as a single chart with
 
 ![Multiple Axis Mode in Blazor PivotChart](images/blazor-pivotchart-multiple-axis-mode.png)
 
+Additionally, to display chart series for multiple values within a single y-axis, set the properties 
+[EnableMultipleAxis](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.PivotView.PivotChartSettings.html#Syncfusion_Blazor_PivotView_PivotChartSettings_EnableMultipleAxis) to **true** and the [MultipleAxisMode](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.PivotView.MultipleAxisMode.html) to **Combined**, in the [PivotChartSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.PivotView.PivotChartSettings.html).
```

Comment:
There is a line break here.

Check in LIVE: https://blazor.syncfusion.com/documentation/pivot-table/pivot-chart